scipy/stats/stats.py

```diff
@@ -8121,6 +8121,281 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
     return BrunnerMunzelResult(wbfn, p)
 
 
+def _confint_lowerbound(n, quantile, confidence):
+    r"""
+    Compute the lower bound for a one-sided confidence interval
+    for a given
+    - quantile (0<`quantile`<1)
+    - confidence level (0<`confidence`<1)
+    - number of samples `n`.
+
+    Returns the largest index of the sample being a valid lower bound,
+    or `None` if there are not enough samples to derive one.
+
+    Used by the public function confint_quantile().
+
+    .. versionadded:: 1.7.0
+    """
+
+    # compute all probabilities from the binomial distribution for
+    # the quantile of interest
+    bd = distributions.binom(n, quantile)
+
+    # the lower bound is the last index before the invert survival
```

Comment:
```suggestion
    # the lower bound is the last index before the inverse survival
```
Since `confidence` is typically close to 1, we are in the left tail of the distribution, so it is more natural for me to think of `bd.ppf(1-confidence)` than `bd.isf(confidence)`. But I suppose this avoids subtractive cancellation when `confidence` is very close to `1`.